public_html/i18n/en.json

```diff
@@ -0,0 +1,71 @@
+{
+  "@metadata": {
+    "authors": [
+      "Niharika Kohli",
+      "MusikAnimal",
+      "Ryan Kaldari"
+    ]
+  },
+  "name": "CopyPatrol",
+  "documentation": "Documentation",
+  "feedback": "Feedback",
+  "leaderboard": "Leaderboard",
+  "login": "Login",
+  "logout": "Logout",
+  "header-loggedin": "You are logged in as: $1",
+  "form-all": "All cases",
+  "form-open": "Open cases",
+  "form-reviewed": "Reviewed cases",
+  "form-mine": "My reviews",
+  "form-drafts": "Drafts only",
+  "wikiprojects": "Wikiprojects",
+  "form-wikiprojects-placeholder": "Type Wikiproject names...",
+  "submit": "Submit",
+  "page": "Page",
+  "diff": "Diff",
+  "editor": "Editor",
+  "review": "Review",
+  "review-fixed": "Page fixed",
+  "review-fixed-tooltip": "The edit was a copyright violation and has been reverted",
+  "review-false": "No action needed",
+  "review-false-tooltip": "The edit is a false positive, nothing needs to be done",
+  "review-by": "Reviewed by:",
+  "history": "History",
+  "talk": "Talk",
+  "contribs": "Contributions",
+  "record-editcount": "Edit count:",
+  "compare": "Compare",
+  "record-ithenticate": "iThenticate report",
+  "record-url-text": "$1 of edit ($2 words)",
+  "record-noeditor": "No editor found",
+  "record-noeditor-tooltip": "The revision may have been deleted or the data is not yet available in Labs database.",
+  "record-ores": "ORES score: $1",
+  "article": "Article",
+  "compare-article-loading": "Loading article revision...",
+  "compare-source-loading": "Loading possible copyvio source...",
+  "footer-loadmore": "Load More",
+  "footer-nomore": "No more records!",
+  "footer-broughtby": "Brought to you by $1",
+  "footer-commtech": "Community Tech",
+  "footer-poweredby": "Powered by $1 and $2",
+  "source": "Source",
+  "phabricator": "Phabricator",
+  "footer-imageattr": "Image attribution",
+  "lboard-title": "Leaderboard",
+  "lboard-week": "Cases closed in the<br/> last 7 days",
```

Comment:
Hardcoded linebreaks don't really work for translation, since the translators won't know where to put the linebreaks to make sure things wrap correctly. Danny originally wanted this done so that all the columns lined up right, but we can probably figure out a better way to do this in a future commit. For now, let's just remove the line breaks and let the lines wrap naturally (even if they don't all line up perfectly).